scene/main/http_request.cpp

```diff
@@ -272,6 +279,7 @@ bool HTTPRequest::_update_connection() {
 		} break;
 		case HTTPClient::STATUS_CONNECTED: {
 
+			timer->stop();
```

Comment:
Didn't we agree not to stop the timer on connection but leave it as a timeout on the whole request?

Reply:
Sorry, missed that 😳